packages/core/parcel/src/cli.js

```diff
@@ -39,10 +39,18 @@ async function logUncaughtError(e: mixed) {
   await new Promise(resolve => setTimeout(resolve, 100));
 }
 
-process.on('unhandledRejection', async (reason: mixed) => {
```

Comment:
We should probably still keep this, in cases there is some promise anywhere that doesn't have a `catch` handler.

Reply:
It is still there, I just changed it to `process.on('unhandledRejection', handleUncaughtException);`

So I could re-use the logic for the run function as it should basically do the same thing...